lib/danger/commands/local.rb

```diff
@@ -0,0 +1,35 @@
+module Danger
+  class Local < Runner
+    self.description = 'Run the Dangerfile locally.'
+    self.command = 'local'
+
+    def initialize(argv)
+      @dangerfile_path = "Dangerfile" if File.exist? "Dangerfile"
+      super
+    end
+
+    def validate!
+      super
+      unless @dangerfile_path
+        help! "Could not find a Dangerfile."
+      end
+    end
+
+    def run
+      # The order of the following commands is *really* important
+      dm = Dangerfile.new
+      dm.env = EnvironmentManager.new(ENV)
+      dm.env.ci_source = 
+      dm.env.fill_environment_vars
```

Comment:
Indent the first parameter one step more than the start of the previous line.<br>Indent the first line of the right-hand-side of a multi-line assignment.